packages/invoices/src/InboundShipment/ListView/ListView.tsx

```diff
@@ -1,3 +1,138 @@
-export const ListView = () => {
-  return null;
+import React, { FC, useState } from 'react';
+import { useNavigate } from 'react-router';
+
+import {
+  DataTable,
+  useColumns,
+  InvoiceRow,
+  useListData,
+  getNameAndColorColumn,
+  TableProvider,
+  createTableStore,
+  Color,
+  useOmSupplyApi,
+  useNotification,
+  generateUUID,
+  RouteBuilder,
+} from '@openmsupply-client/common';
+import { getInboundShipmentListViewApi } from './api';
+import { NameSearchModal } from '@openmsupply-client/system/src/Name';
+import { Toolbar } from './Toolbar';
+import { AppBarButtons } from './AppBarButtons';
+import { AppRoute } from '@openmsupply-client/config';
+
+export const InboundListView: FC = () => {
+  const navigate = useNavigate();
+  const { error } = useNotification();
+  const { api } = useOmSupplyApi();
+
+  const {
```

Comment:
again another massive copy and paste. I changed a couple things from outbound to inbound